examples/siro_sandbox/configs/base_rearrange_humanoid_spot.yaml

```diff
@@ -64,7 +64,7 @@ habitat:
         arm_controller: "ArmRelPosMaskKinematicAction"
         arm_joint_mask: [1,1,0,1,1,1,1]
         arm_joint_dimensionality: 7
-        grasp_thresh_dist: 0.7
+        grasp_thresh_dist: 0.475 # Eric selects number 0.25; the original value is 0.7
```

Comment:
Hi Eric, I changed the grasp_thresh_dist to be a bit generous. We can turn this parameter later